client/components-v1/ForgingModal.jsx

```diff
@@ -61,12 +71,14 @@ const ForgingModal = ({ open, onClose, onSave }) => {
   if (!open) return null;
 
   return (
-    <div className='overlay'>
-      <div className='modalContainer'>
-        <button className='closeBtn' onClick={onClose}>x</button>
+    <div className="overlay">
```

Comment:
we are using modulated SCSS, these classes wont work properly...

Reply:
these were pulled out of old components, my fault.

Reply:
actualy ignore all of the v1s because they were added to PR because of prettier. they arent used, ignore